test/test.html

```diff
@@ -6,11 +6,13 @@
   <div id="qunit"></div>
   <div id="qunit-fixture"></div>
   <script src="https://code.jquery.com/qunit/qunit-2.19.4.js"></script>
+
   <!-- Include test modules below. -->
   
-<!--  <script src="temp-test.js" type="module" defer></script>-->
+  <script src="temp-test.js" type="module" defer></script>
```

Comment:
Make sure that the script tag that includes `temp-test.js` on line `12` is removed in this pull request or your other pull request (#66) before merging the branch. Otherwise, looks great!